client/assets/js/admin-controller.js

```diff
@@ -137,5 +145,148 @@ angular.module('portal').controller('adminController', function ($scope, $http,
             // needs implementation
         });
     };
-    
+
+    // get all admin
+    $http.get('/api/user/getUsers').then(function(response) {
+        $scope.admins = {};
+        $scope.comUsers = {};
+        $scope.userData = response.data;
+        console.log("ALL USERS: ", $scope.userData);
+        $scope.userData.forEach(function(user) {
+            if (user.isAdmin === true) {
+                $scope.admins[user.email] = user.name;
+            } else {
+                $scope.comUsers[user.email] = user.name;
+            };
+        });
+    });
+
+
+    $scope.addAdmin = function() {
+        $scope.internalError = false;
+
+        if ($scope.comUsers[$scope.changeAdminEmail]) {
+
+            $scope.changeUser = $http.post('/api/user/getSingleUser', {
+                "email": $scope.changeAdminEmail
+            }).then(function (response) {
+                console.log("Found User!")
+            }, function (response) {
+                $scope.internalError = true;
+                console.log('Error finding user.');
+            });
+
+            $scope.userData.forEach(function(user) {
```

Comment:
i think the checks to see if the email is valid should be done by the backend, not in the frontend